lib/hunter/commands/hunt.rb

```diff
@@ -173,22 +174,30 @@ def process_packet(data:)
 
         EOF
 
+        buffer = NodeList.load(Config.node_buffer)
+        parsed = NodeList.load(Config.node_list)
+        dest = buffer
+        if node.hostname.match(Regexp.new(@auto_regex))
```

Comment:
Does `Regexp.new(@auto_regex)` behave differently if I provide a regular expression with or without the following/preceding `/`?

Reply:
It's the same implementation as is used in `remove-node`'s `--name` option - I believe that using `/node/` as your argument would indeed attempt to match the string `/node/`, with the `/`s being taken as standard characters